solitude/filter.py

```diff
@@ -10,6 +13,16 @@ class StrictQueryFilter(DjangoFilterBackend):
     Instead limit it down to the parameters allowed in filter_fields.
     """
 
+    def get_filter_class(self, view, queryset=None):
+        klass = (super(StrictQueryFilter, self)
+                 .get_filter_class(view, queryset=queryset))
+        try:
+            # If an ordering exists on the model, use that.
+            klass._meta.order_by = klass.Meta.model.Meta.ordering
```

Comment:
could this just be `hasattr(klass.Meta.model.Meta, 'ordering')` ? Or could it be missing `.model` too?